modules/really-core/src/test/scala/io/really/protocol/ResponseWritesSpec.scala

```diff
@@ -119,7 +119,6 @@ class ResponseWritesSpec extends BaseActorSpec {
     val obj = Json.toJson(response)
 
     assertResult(Json.obj(
-      "meta" -> Json.obj("subscription" -> JsNull),
```

Comment:
I don't think this is enough testing, would you please update the rest of test cases to validate that nulls are not really generated anymore?